app/scripts/views/mixins/account-locked-mixin.js

```diff
@@ -0,0 +1,61 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * Handle locked accounts. Mixed into views.
+ *
+ * @class AccountLockedMixin
+ */
+'use strict';
+
+define([
+  'views/base',
+  'lib/auth-errors'
+], function (BaseView, AuthErrors) {
+  var t = BaseView.t;
+
+  var AccountLockedMixin = {
+    events: {
+      'click a[href="/confirm_account_unlock"]':
+            BaseView.preventDefaultThen('sendAccountLockedEmail')
+    },
+
+    notifyOfLockedAccount: function (account) {
+      this._lockedAccount = account;
+
+      var err = AuthErrors.toError('ACCOUNT_LOCKED');
+      err.forceMessage = t('Account locked. <a href="/confirm_account_unlock">Send unlock email</a>');
```

Comment:
Ah, interesting, perhaps this already addresses my comments about "send" versus "resend" from earlier.  `forceMessage` sounds pretty brutal...